domain_alias/src/DomainAliasForm.php

```diff
@@ -89,6 +89,7 @@ public function save(array $form, FormStateInterface $form_state) {
     else {
       drupal_set_message($this->t('Domain alias updated.'));
     }
+    \Drupal\Core\Cache\Cache::invalidateTags(array('rendered'));
```

Comment:
I like to have a `use` declaration at the top if we're going to call a static. But I thing this should be called from the service handler, not by directly calling the class.

e.g.

`\Drupal::services('cache')`